package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu May 17 13:32:37 UTC 2018 - jlopez@suse.com
+
+- Allow to consider MD RAIDs as BIOS RAIDs by using the env
+  variable LIBSTORAGE_MDPART (bsc#1092417).
```

Comment:
please also document variable somewhere. IDeally in suse-doc and in release notes.

Reply:
This value is not new, we are bringing back it. Is not already documented?

Reply:
Sorry, zero results found with google neither at suse-doc. So we should definitively document it. 
Either installation guide or storage guide.